Text/Megaparsec/Error.hs

```diff
@@ -248,6 +252,98 @@ parseErrorPretty
 parseErrorPretty e =
   sourcePosStackPretty (errorPos e) <> ":\n" <> parseErrorTextPretty e
 
+
+-- | Pretty-print a 'ParseError' and display the line on which the parse error
+-- occurred. The rendered 'String' always ends with a newline.
+--
+-- Assumes that the stream tokens can be coerced to an @Int@ and that the value
```

Comment:
I think "converted" is a better word to use here, as "coerced" implies that we use actual coercion between types that is supported by GHC (`coerce` for type-safe coercion and `unsafeCoerce` for unsafe coercion), which is not the case.

You could also mention `fromEnum`, so it's clear how the "conversion" is performed.